src/Files.App/UserControls/StatusBarControl.xaml

```diff
@@ -350,9 +351,32 @@
 							Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
 							IsItemClickEnabled="True"
 							ItemClick="BranchesList_ItemClick"
-							ItemsSource="{x:Bind DirectoryPropertiesViewModel.BranchesNames, Mode=OneWay}"
+							ItemsSource="{x:Bind DirectoryPropertiesViewModel.Branches, Mode=OneWay}"
 							SelectedIndex="{x:Bind DirectoryPropertiesViewModel.SelectedBranchIndex, Mode=TwoWay}"
-							SelectionMode="Single" />
+							SelectionMode="Single">
+
+							<ListView.ItemTemplate>
+								<DataTemplate x:DataType="data:BranchItem">
+									<Grid HorizontalAlignment="Stretch">
+										<Grid.ColumnDefinitions>
+											<ColumnDefinition />
+											<ColumnDefinition Width="Auto" />
+										</Grid.ColumnDefinitions>
+
+										<TextBlock
+											VerticalAlignment="Center"
+											Text="{x:Bind Name}"
+											TextTrimming="CharacterEllipsis" />
+										<Button
+											Grid.Column="1"
+											Click="DeleteBranch_Click"
+											Content="{helpers:ResourceString Name=Delete}"
```

Comment:
What do you think about using an icon instead?

Reply:
Good idea, I think it would look better! Something like a red bin?

Reply:
I'll let @mdtauk answer about the color, but as far as the icon goes, a bin is probably a good choice.